crates/walrus-service/src/node/server/routes.rs

```diff
@@ -80,7 +87,9 @@ pub const DELETABLE_BLOB_CONFIRMATION_ENDPOINT: &str =
 /// The path to get recovery symbols.
 pub const RECOVERY_SYMBOL_ENDPOINT: &str = "/v1/blobs/{blob_id}/recoverySymbols/{symbol_id}";
 /// The path to get multiple recovery symbols.
-pub const RECOVERY_SYMBOL_LIST_ENDPOINT: &str = "/v1/blobs/{blob_id}/recoverySymbols";
+pub const LIST_RECOVERY_SYMBOL_ENDPOINT: &str = "/v1/blobs/{blob_id}/recoverySymbols";
+/// The path to get raw recovery symbols without proof.
+pub const LIST_RAW_RECOVERY_SYMBOL_ENDPOINT: &str = "/v1/blobs/{blob_id}/rawRecoverySymbols";
```

Comment:
Should we also make nodes return proof along with the decoding symbol? Otherwise, how can we know which node(s) are returning garbage?